daemon/execdriver/windows/exec.go

```diff
@@ -13,13 +13,14 @@ import (
 	"github.com/natefinch/npipe"
 )
 
-func (d *driver) Exec(c *execdriver.Command, processConfig *execdriver.ProcessConfig, pipes *execdriver.Pipes, startCallback execdriver.StartCallback) (int, error) {
+// Exec implements the exec driver Driver interface.
```

Comment:
@hqhq you should describe what Exec actually does. The fact that implements the Driver interface is a good additional piece of information but secondary.